packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -108,8 +108,8 @@ export class TSClient implements Generatable {
         ? (Object.keys(binaryPaths.libqueryEngine ?? {}) as BinaryTarget[])
         : (Object.keys(binaryPaths.queryEngine ?? {}) as BinaryTarget[])
 
-    const inlineSchema = (await readFile(schemaPath)).toString('base64')
-    const inlineSchemaHash = crypto.createHash('sha256').update(inlineSchema).digest('hex')
+    const inlineSchema = await readFile(schemaPath, 'utf8')
+    const inlineSchemaHash = crypto.createHash('sha256').update(btoa(inlineSchema)).digest('hex')
```

Comment:
Do we actually need to `btoa` schema here? IIRC, accelerate treats hash as opaque, so sending hash of a textual schema might work as well.

Reply:
That was not the case at least with miniproxy, so I went with this.